packages/core/src/components/ModalNew/layouts/ModalBasicLayout/ModalBasicLayout.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { forwardRef } from "react";
+import cx from "classnames";
+import { getTestId } from "../../../../tests/test-ids-utils";
+import { ComponentDefaultTestId } from "../../../../tests/constants";
+import styles from "./ModalBasicLayout.module.scss";
+import { ModalBasicLayoutProps } from "./ModalBasicLayout.types";
+import { useModal } from "../../context/ModalContext";
+import Flex from "../../../Flex/Flex";
+import Divider from "../../../Divider/Divider";
+
+const ModalBasicLayout = forwardRef(
+  (
+    { children, className, id, "data-testid": dataTestId }: ModalBasicLayoutProps,
+    ref: React.ForwardedRef<HTMLDivElement>
+  ) => {
+    const [header, content] = React.Children.toArray(children);
+    const { contentScrolled } = useModal();
```

Comment:
Should be `isContentScrolled`?

Reply:
I was wondering about that one, I always get wondering what is a better practice

Reply:
On base elements it should be as close to native attrs IMO, other than that - if it's not intuitive enough (like disabled) than it should imply the type

Reply:
not sure I'm following

Reply:
`<Button disabled>` vs `<ModalAction isContentScrolled>`

Reply:
got ya
will change then 🙏🏼